students/biegon_piotr/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,56 @@
+print("Welcome to the awesome calculator :)")
+
+
+def options():
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h, ? - help")
+    print("q - QUIT")
+
+
+def data():
+    var_1 = float(input("Enter the first number: "))
+    var_2 = float(input("Enter the second number: "))
+    return var_1, var_2
+
+
+while True:
+    print("\nAvailable options:")
+    options()
+    option = input("\nChoose one option: ")
+
+    if option == "a":
+        print("\nADDING")
+        var_1, var_2 = data()
+        print("Addition result: {:f}".format(var_1 + var_2))
```

Comment:
The idea was to put every action into separate method.
So in the main loop would like more like this:
```
       if option == "a":
            add()
```